Views/Items/Details.cshtml

```diff
@@ -29,14 +29,14 @@
             @Html.DisplayFor(model => model.Nutrients.Calories)
         </dd>
         <dt class="col-sm-2">
-            @Html.DisplayNameFor(model => model.Nutrients.ItemID)
```

Comment:
It's usually best to use more simple types in the view model. Also, surely you could use `model.Id` or something instead of `model.Nutrients.ItemId`?